eth/peer.go

```diff
@@ -139,6 +202,19 @@ func (p *peer) SendTransactions(txs types.Transactions) error {
 	return p2p.Send(p.rw, TxMsg, txs)
 }
 
+// AsyncSendTransactions queues list of transactions propagation to a remote
+// peer. If the peer's broadcast queue is full, the event is silently dropped.
+func (p *peer) AsyncSendTransactions(txs []*types.Transaction) {
+	select {
+	case p.queuedTxs <- txs:
```

Comment:
The limit of 128 transactions on the channel can get full pretty quickly, no? Won't that mean that we'll  be stuck here?

Reply:
There's a `default` clause that drops anything that doesn't fit.